src/BinSkim.Driver/RoslynAnalyzer/ILImporter.Driver.cs

```diff
@@ -0,0 +1,915 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// Source: https://github.com/dotnet/corert/blob/master/src/Common/src/TypeSystem/IL/ILImporter.cs
```

Comment:
It's lifted from https://github.com/dotnet/corert. After some initial stabs at it, I realized, I was reinventing the lowest-level part and that corert had a nicely factored pattern where you can fill in the domain-specific part of the importer partial class. We'll have to figure out how/if to share this code by means other than copy/paste in the medium term (e.g. possibly by pushing down some things to S.R.M).